src/components/Todo-Add.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div>
+        <input type="text" placeholder="Add Title..."  autofocus v-model="newTodo.title" maxlength="24" class="todo-title">
+        <input type="text"  placeholder="Add Note..." v-model="newTodo.notes" maxlength="28" @keyup.enter="addTodo(newTodo)" :disabled="!newTodo.title" class="todo-notes">
+    </div>
+</template>
+
+<script>
+  export default {
+    name: 'todoAdd',
+    data() {
+      return {
+        newTodo: {
+          title: '',
+          notes: '',
+          completed: false
+        }
+      }
+    },
+    methods: {
+      addTodo() {
+        let todoCopy = JSON.parse(JSON.stringify(this.newTodo));
```

Comment:
Interesting.... 🤔 

How about if we change the data to strings instead of a object? Then we don't have to deal with parsing and stringifying the object getting passed around. It'll be a clean array.

What do you think?

new data 👇 
```javascript
data() {
      return {
        title: '',
        notes: ''
      }
    }
```
new method 👇 
```javascript
addTodo() {
        this.$emit('todoAdded', this.title, this.notes);
        this.title = '';
        this.notes = '';
      }
```

new method on `Todo-List.vue`
```javascript
methods: {
    addTodo(title, notes) {
      this.todos.push( { title, notes, completed: false });
    }
  }
```

You'll have to adjust the inputs too ❤️ 
```javascript
<input type="text" placeholder="Add Title..."  autofocus v-model="title" maxlength="24" class="todo-title">
<input type="text"  placeholder="Add Note..." v-model="notes" maxlength="28" @keyup.enter="addTodo" :disabled="!title" class="todo-notes">
```